src/main/kotlin/io/polygon/kotlin/sdk/rest/experimental/Events.kt

```diff
@@ -0,0 +1,66 @@
+package io.polygon.kotlin.sdk.rest.experimental
+
+import com.thinkinglogic.builder.annotation.Builder
+import io.ktor.http.*
+import io.polygon.kotlin.sdk.rest.ComparisonQueryFilterParameters
+import io.polygon.kotlin.sdk.rest.applyComparisonQueryFilterParameters
+import io.polygon.kotlin.sdk.rest.Paginatable
+import io.polygon.kotlin.sdk.rest.PolygonRestOption
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@SafeVarargs
+@ExperimentalAPI
+suspend fun PolygonExperimentalClient.getTickerEvents(params: TickerEventsParameters, vararg opts: PolygonRestOption): TickerEventsResponse =
+    polygonClient.fetchResult({
+        path("vX", "reference", "tickers", params.id, "events")
```

Comment:
Since `id` is a path param it's effectively required, we should make it an explicit parameter for this function instead of putting it inside a params class. That way a user can't accidentally omit the ID param

Reply:
Fixed.